integration_tests/test_ica_precompile.py

```diff
@@ -263,4 +265,22 @@ def submit_msgs_ro(func, str):
     wait_for_status_change(tcontract, last_seq)
     status = tcontract.caller.statusMap(last_seq)
     assert expected_seq == last_seq
-    assert status == Status.TIMEOUT
+    assert status == Status.FAIL
+
+    # balance and seq should not change on fail
+    str = submit_msgs(
+        ibc,
+        tcontract.functions.callSubmitMsgs,
+        data,
+        ica_address,
+        False,
+        expected_seq,
```

Comment:
seq don't increase?

Reply:
good catch, seems need test timeout case at last to avoid channel get inactive